src/docs/reference/modules/Localize/README.md

```diff
@@ -166,16 +166,3 @@ Then, you will be able to run this command to generate the .po files:
 ``` bash
 extractpo <INTPUT_PATH> <OUTPUT_PATH> [-l|--language {"C#"|"VB"}] [-t|--template {"razor"|"liquid"}]
 ```
-
-### Shapes
-
-#### `AdminCulturePicker`
```

Comment:
This is behaviour breaking change

Reply:
@hishamco how is this a breaking change?

Reply:
`ContentCulturePicker` & `AdminCulturePicker` have been used for awhile, removing them will breaks an existing apps

Reply:
we did not remove `ContentCulturePicker` or `AdminCulturePicker`. In a different PR we removed `ContentCulturePickerContainer` and `AdminCulturePickerContainer` which did nothing more than creating a the actual shapes `ContentCulturePicker` and `AdminCulturePicker`. It would me much more helpful to actually test the changes than going back and fourth with the details.